src/data/faq_de.json

```diff
@@ -102,6 +102,159 @@
+                        ]
+                    },
+                    {
+                        "title": "Kann man auch im digitalen Impfnachweis speichern, dass man bereits infiziert war oder negativ getestet wurde?",
+                        "anchor": "vaccination_certificate_recovered_negative_test",
+                        "active": true,
+                        "textblock": [
+                            "Ja, auch negative Tests oder eine durchgemachte Infektion lassen sich in der CovPass- App und Corona-Warn-App als Testzertifikat bzw. Genesenenzertifikat hinterlegen. Eine durchgemachte Infektion wird mittels eines PCR-Test-Ergebnisses dokumentiert."
+                        ]
+                    },
+                    {
+                        "title": "Warum wurde kein EU-weites Projekt für ein Zertifikat ausgeschrieben?",
+                        "anchor": "vaccination_certificate_eu",
+                        "active": true,
+                        "textblock": [
+                            "Eine gemeinsame EU-Ausschreibung hätte zu viel Zeit benötigt und wäre aufgrund der unterschiedlichen Impfinformationssysteme in den Mitgliedstaaten auch schwierig umzusetzen gewesen. Beim EU-Ansatz geht es um die Regelung eines Anerkennungsrahmens. Etwaige grenzüberschreitende Nutzungspiloten könnten seitens der EU-Kommission begleitet werden. Bei der Umsetzung des Digitalen Impfnachweises in Deutschland werden die EU-Vorgaben von vornherein berücksichtigt."
+                        ]
+                    },
+                    {
+                        "title": "Wirkt sich der Impfstatus auf die Berechnung des Risikos aus? Beispielsweise, wenn eine geimpfte Person sich infiziert haben sollte und dann andere Nutzer*innen warnt?",
+                        "anchor": "vaccination_certificate_risk_assesment",
+                        "active": true,
+                        "textblock": [
+                            "Nein, in der Risikoberechnung wird nicht berücksichtig, ob Sie gegen Corona geimpft oder genesen sind. Die Funktionen sind voneinander unabhängig. Die Qualität Ihrer Risikobewertung ändert sich nicht, je nachdem, ob Sie Sich dafür entscheiden, Ihren Impfausweis in der Anwendung zu speichern, was unterstreicht, dass dies völlig freiwillig ist. Ihr persönlicher Impfstatus kann jedoch Ihre Interpretation der Risiko-Warnung beeinflussen."
+                        ]
+                    },
+                    {
+                        "title": "Kann ich mein Impfzertifikat auf mehreren Smartphones speichern, z.B. auf meinem privaten und dem Geschäftshandy? Wie kann ich die Impfbescheinigung beim Gerätewechsel übertragen? Was mache ich, wenn ich mein Smartphone verloren habe? Ist das Impfzertifikat in meinem Backup enthalten? Kann ich das Zertifikat in der Corona-Warn-App und der CovPass-App gleichzeitig verwalten, oder muss ich mich für eine der beiden Apps entscheiden?",

```

Comment:
"auf meinem privaten und dem Geschäftshandy" not sure whether this is good grammar, as I suggested before.

Reply:
agreed.

Reply:
fixed